src/Microsoft.ApiDesignGuidelines.Analyzers/Core/OverrideEqualsOnOverloadingOperatorEquals.Fixer.cs

```diff
@@ -27,10 +23,57 @@ public sealed override FixAllProvider GetFixAllProvider()
         }
 
         public sealed override async Task RegisterCodeFixesAsync(CodeFixContext context)
```

Comment:
The more I think about it, the more it feels like a full-fledged story.  As soon as you have either operator, we should offer to add the other one, Equals, GetHashCode, and possibly IEquatable (for structs).  We might also want to tie into the smarter Equals and GetHashCode generators that exist as refactorings and generate boilerplate from specified fields (and, now, readonly autoproperties?).